packages/main/src/plugin/extension-loader.ts

```diff
@@ -1079,49 +1080,101 @@ export class ExtensionLoader {
     const navigate: typeof containerDesktopAPI.navigate = {
       container: {
         viewContainers: async function (): Promise<void> {
-          apiSender.send('navigate', '/containers');
+          apiSender.send('navigate', {
+            page: Page.CONTAINERS,
+          } as NavigateRequest);
```

Comment:
wondering why we add `as NavigateRequest` as it's basically casting any object to be NavigateRequest even if the object is not compliant at all
If you're searching for type-safety, I would argue that you should call a function taking a NavigateRequest parameter that do the apiSender.send('navigate') instead of calling apiSender.send('navigate' everywhere in extension-loader.js so you've type-safety on calls and don't need the usage of `as NavigateRequest`

Reply:
Thanks for the tips, I haven't thought directly about this solution!